sros2/test/sros2/commands/security/verbs/test_create_enclave.py

```diff
@@ -94,10 +96,26 @@ def test_create_key(enclave_keys_dir):
         assert (enclave_keys_dir / expected_file).is_file()
 
 
+def test_create_enclave_twice(tmpdir):
+    keystore_dir = Path(tmpdir)
+
+    # First, create the keystore
+    sros2.keystore.create_keystore(keystore_dir)
+    assert keystore_dir.is_dir()
+
+    # Now using that keystore, create an enclave
+    assert cli.main(
+        argv=['security', 'create_enclave', str(keystore_dir), '/test_enclave']) == 0
+    enclave_dir = keystore_dir / 'enclaves' / 'test_enclave'
+    assert enclave_dir.is_dir()
+
+    # Now create it again and confirm... something
```

Comment:
Is this a TODO ?

Reply:
Uh, yeah, totally forgot to add that last assertion :blush: . Fixed.